docs/manual/cluster-admin/how-to-set-up-docker-image-cache.md

```diff
@@ -0,0 +1,104 @@
+* `docker_azure_account_name`: required when storage backend is "azure", should be your azure blob storage accountname.
+* `docker_azure_account_key`: required when storage backend is "azure", should be your azure blob storage base64 encoded account key.
+* `docker_cache_azure_container_name`: required when storage backend is "azure", should be modified if you want to specify container name your docker-cache use, default is "dockerregistry".
+* `docker_cache_fs_mount_path`: required when storage backend is "filesystem", should be modified if you want to specify path your docker-cache use, default is "/var/lib/registry".
+* `docker_cache_remote_url`: pull-through cache remote url, should be modified if you want to specify other docker remote registry rather than dockerhub, default is "https://registry-1.docker.io/".
+* `docker_cache_htpasswd`: htpasswd auth info with base64 encoded, should be use with ssl when docker-cache cache some private registry as an access control method.
+
+### `config.yaml` example with azure
+
+``` yaml
+user: forexample
+password: forexample
+docker_image_tag: v1.5.0
+
+# Optional
+
+#######################################################################
+#                    OpenPAI Customized Settings                      #
+#######################################################################
+# enable_hived_scheduler: true
+enable_docker_cache: true
+docker_cache_storage_backend: "azure"
+docker_cache_azure_account_name: "forexample"
+docker_cache_azure_account_key: "forexample"
+# docker_cache_azure_container_name: "dockerregistry"
+# docker_cache_fs_mount_path: "/var/lib/registry"
+# docker_cache_remote_url: "https://registry-1.docker.io"
+# docker_cache_htpasswd: "" 
+# enable_marketplace: "true"
+
+# ...
+
+```
+
+Make sure the setting of `enable_docker_cache` was `"true"` (include the quotation marks), and finish the [installation](./installation-guide.md), the marketplace will be set up.
+
+### `config.yaml` example with filesystem
+
+``` yaml
+user: forexample
+password: forexample
+docker_image_tag: v1.5.0
+
+# Optional
+
+#######################################################################
+#                    OpenPAI Customized Settings                      #
+#######################################################################
+# enable_hived_scheduler: true
+enable_docker_cache: true
+docker_cache_storage_backend: "filesystem"
+# docker_cache_azure_account_name: ""
+# docker_cache_azure_account_key: ""
+# docker_cache_azure_container_name: "dockerregistry"
+docker_cache_fs_mount_path: "/var/lib/registry"
+# docker_cache_remote_url: "https://registry-1.docker.io"
+# docker_cache_htpasswd: "" 
+# enable_marketplace: "true"
+
+# ...
+
+```
+
+Make sure the setting of `enable_docker_cache` was `"true"` (include the quotation marks), and finish the [installation](./installation-guide.md), the marketplace will be set up.
+
```

Comment:
Please also add changes to `config.yaml` to the installation guide:

https://github.com/microsoft/pai/blob/master/docs/manual/cluster-admin/installation-guide.md

Also, please add a link in installation-guide.md to let users aware of this doc.